playbooks/cluster/openshift/vars/gluster.yml

```diff
@@ -0,0 +1,11 @@
+# Namespace to create for the GlusterFS and heketi resources
+openshift_storage_glusterfs_namespace: app-storage
+# Do not create a StorageClass; a custom StorageClass will be created via APB
+# for use by kubevirt
+openshift_storage_glusterfs_storageclass: false
+# glusterblock functionality is not supported outside of Logging/Metrics
+openshift_storage_glusterfs_block_deploy: false
```

Comment:
I don't follow this one too. May I set it to `true`? Will it be supported then?

Reply:
No. Upstream, glusterblock support is enabled by default (true). Downstream, we only support the use of glusterblock for Logging and Metrics. Since this project is not using glusterblock for Logging and Metrics, we disable it.